mlflow/tracking/_tracking_service/utils.py

```diff
@@ -95,10 +95,13 @@ def _resolve_tracking_uri(tracking_uri=None):
     return tracking_uri or get_tracking_uri()
 
 
-def get_tracking_uri() -> str:
+def get_tracking_uri(tracking_uri:str|None=None) -> str:
```

Comment:
This syntax is only supported in Python 3.10 and newer. 
Could you try to specify this in the format that is supported for Python 3.8 (minimum supported version of Python from MLflow and what our CI system uses to validate our tests?) 

```python
from typing import Union

def get_tracking_uri(tracking_uri: Union[str, None] = None) -> str:
```